itests/hcatalog-unit/src/test/java/org/apache/hive/hcatalog/listener/TestDbNotificationListener.java

```diff
@@ -355,6 +374,7 @@ public void dropDatabase() throws Exception {
 
     // Two events: one for create db and other for drop db
     assertEquals(2, rsp.getEventsSize());
+    testEventCounts(dbName, firstEventId, null, null, 2);
```

Comment:
Shall test with different cases for db names.

Reply:
There are tests with other database name e.g. the one on line 408. Would that suffice?

Reply:
I think all of them uses lower case. Please check again. If there are db and table names with different case, then that's fine.

Reply:
I think all of the db names are lower cases. Please check again if not there, then add some.

Reply:
TestReplicationScenariosAcidTable has a case where the database name has letters with both the case and is passed to getNotificationEventsCount() to as is. Per our offline discussion fixed getNotificationEventsCount() so that it converts the database name into lower case if it's not * pattern, in which case the condition on database name is excluded from the query.